Resources/views/Ticket/show_attachment.html.twig

```diff
@@ -1,3 +1,4 @@
+{% import 'HackzillaTicketBundle:Macros:macros.html.twig' as macros %}
```

Comment:
ec233c90bb2cdba896031bae541f51b157694abb is now cherry-picked into `master`, thanks for pointing the issue.
```suggestion
{% import '@HackzillaTicket/Macros/macros.html.twig' as macros %}
```